tests/impl.cpp

```diff
@@ -11797,8 +11829,15 @@ result_t test_mm_set_denormals_zero_mode(const SSE2NEONTestImpl &impl,
 result_t test_rdtsc(const SSE2NEONTestImpl &impl, uint32_t iter)
 {
     uint64_t start = _rdtsc();
-    for (int i = 0; i < 100000; i++)
+    for (int i = 0; i < 100000; i++) {
+#if defined(_MSC_VER)
+        __nop();
```

Comment:
These nops are left over from me messing around with preventing the optimizer from eliminating this loop. The _ReadWriteBarrier does the trick and is probably the most robust; the nops can go.

Reply:
ACK, will remove next commit